py/picca/data.py

```diff
@@ -958,6 +958,7 @@ class Delta(QSO):
             Neighbouring deltas/quasars
         fname: string or None
             String identifying Delta as part of a group
+        delta_log_lambda
```

Comment:
add explanation and type

Reply:
Sorry I copied this line when it was already there.